apps/iiif/serializers/canvas.py

```diff
@@ -82,6 +86,21 @@ def start_object(self, obj):
     def get_dump_object(self, obj):
         obj.label = str(obj.position)
         if ((self.version == 'v2') or (self.version is None)):
+            otherContent = [ 
```

Comment:
I noticed this "other content" in the manifest. When Mirador loads annotations from the manifest, it marks them as not editable. Mirador loads user annotations separately. I added a check in Mirador that will not load annotations from the manifests where the value for "annotated by" matches the current user. Maybe it would be better check the label? I trust and agree including the user annotations is proper. I guess I wish Mirador only had one place/way that it loads annotations.